README.md

```diff
@@ -419,3 +419,12 @@ Release plan can also be deleted with
 ```shell
 forklift delete releaseplan 1.0.1 --cluster 7 --application 6 --service 5
 ```
+
+## Release a new version
+
+Update `cmd/root.go` with the new version and create a new tag with
+
+```shell
+git tag <version>
+git push origin --tags
```

Comment:
Git tag creation is part of `release.sh`. I think we should mention it in the readme.

Reply:
done